packages/design-system/src/custom/_shadow.scss

```diff
@@ -0,0 +1,28 @@
+[data-bs-theme="dark"] {
+  --shadow-color: #{if(
+      variable-exists(box-shadow-color-dark),
+      $box-shadow-color-dark,
+      rgba(127, 125, 125, 0.406)
+    )};
+  .shadow-sm {
+    box-shadow:
+      0 1px 2px var(--shadow-color),
+      0 1px 5px var(--shadow-color) !important;
```

Comment:
Can these use the `rem` values that are currently used instead of computed values?
```scss
$box-shadow:                  0 .5rem 1rem rgba($black, .15) !default;
$box-shadow-sm:               0 .125rem .25rem rgba($black, .075) !default;
$box-shadow-lg:               0 1rem 3rem rgba($black, .175) !default;
```